src/index.ml

```diff
@@ -179,36 +210,11 @@ struct
     Semaphore.with_acquire "flush" t.rename_lock (fun () ->
         flush_instance ?no_callback ~with_fsync t)
 
-  module IOArray = Io_array.Make (IO) (Entry)
-
-  module Search =
-    Search.Make (Entry) (IOArray)
-      (struct
-        type t = int
-
-        module Entry = Entry
-
-        let compare : int -> int -> int = compare
-
-        let of_entry e = e.Entry.key_hash
-
-        let of_key = K.hash
-
-        let linear_interpolate ~low:(low_index, low_metric)
-            ~high:(high_index, high_metric) key_metric =
-          let low_in = float_of_int low_metric in
-          let high_in = float_of_int high_metric in
-          let target_in = float_of_int key_metric in
-          let low_out = Int64.to_float low_index in
-          let high_out = Int64.to_float high_index in
-          (* Fractional position of [target_in] along the line from [low_in] to [high_in] *)
-          let proportion = (target_in -. low_in) /. (high_in -. low_in) in
-          (* Convert fractional position to position in output space *)
-          let position = low_out +. (proportion *. (high_out -. low_out)) in
-          let rounded = ceil (position -. 0.5) +. 0.5 in
-          Int64.of_float rounded
-      end)
+  (** {1 RO instances syncing} *)
 
+  (** Loads the log file at [path], if it exists. Used by RO instances, when
+      they have been created before their RW counterpart, to fill the [log]
+      field, or to load the temporary [log_async]. *)
```

Comment:
I think it loads the log_async regardless of when RO is created, but the phrase suggests otherwise. can you rephrase?